bindings/java/native/src/classes/message.rs

```diff
@@ -0,0 +1,149 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{cell::RefCell, rc::Rc};
+
+use iota_wallet::{
+    address::AddressWrapper,
+    message::{
+        Message as MessageRust, MessageId, RemainderValueStrategy as RemainderValueStrategyRust,
+        Transfer as TransferRust, TransferBuilder as TransferBuilderRust,
+    },
+};
+
+use crate::bee_types::{IndexationPayload, MessagePayload};
+
+use chrono::prelude::{DateTime, Utc};
+use std::num::NonZeroU64;
+
+pub enum RemainderValueStrategy {
+    ReuseAddress = 1,
+    ChangeAddress = 2,
+}
+
+pub fn remainder_type_enum_to_type(strategy: RemainderValueStrategy) -> RemainderValueStrategyRust {
+    match strategy {
+        RemainderValueStrategy::ReuseAddress => RemainderValueStrategyRust::ReuseAddress,
+        RemainderValueStrategy::ChangeAddress => RemainderValueStrategyRust::ChangeAddress, /* TODO: Add strategy for
```

Comment:
TODO :) (ignore if this will be resolved later)

Reply:
Created separate method: with_remainder_to_account_with_address